files/galaxy/usegalaxy.org/tool_list.yaml

```diff
@@ -0,0 +1,1058 @@
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'cb7cf57397a7'
+- name: 'variant_filtration'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'da6e2503c62d'
+- name: 'variant_eval'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'fbca1c0956d2'
+- name: 'variant_combine'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '1a6e16391727'
+- name: 'variant_apply_recalibration'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '350a4d0d1678'
+- name: 'variant_annotator'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'ae9d0a543e9b'
+- name: 'unified_genotyper'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '66dd4d4c1743'
+- name: 'realigner_target_creator'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '5b8eaae854da'
+- name: 'print_reads'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'e768f4851646'
+- name: 'indel_realigner'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'bb0beda6cf83'
+- name: 'depth_of_coverage'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - 'c3f08370fc82'
+- name: 'count_covariates'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '14e304b70425'
+- name: 'analyze_covariates'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '738bc749f9a3'
+- name: 'table_recalibration'
+  owner: 'devteam'
+  tool_panel_section_id: 'gatk'
+  revisions:
+  - '30e1dd77e99c'
+# Cloud-only tools
+- name: ncbi_sra_toolkit
+  owner: 'matt-shirley'
+  tool_panel_section_id: convert
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: htseq_bams_to_count_matrix
+  owner: fubar
+  tool_panel_section_id: ngs-rna-tools
+  tool_shed_url: https://testtoolshed.g2.bx.psu.edu
+# Data Managers
+- name: data_manager_snpeff
+  owner: iuc
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_bwa_index_builder
+  owner: devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_gatk_picard_index_builder
+  owner: devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_sam_fasta_index_builder
+  owner: devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_fetch_genome_dbkeys_all_fasta
+  owner: devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_fetch_genome_all_fasta
+  owner: devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_rsync_g2
+  owner: blankenberg
+  tool_shed_url: https://testtoolshed.g2.bx.psu.edu
+- name: data_manager_bowtie2_index_builder
+  owner:  devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_bwa_mem_index_builder
+  owner: devteam
+  tool_shed_url: https://toolshed.g2.bx.psu.edu
+- name: data_manager_picard_index_builder
+  owner: devteam
+  tool_shed_url: https://testtoolshed.g2.bx.psu.edu
```

Comment:
This is also available from the MTS.
Imho we should have a look at all TTS tools and migrate them/fix them.